agent/xds/testdata/listeners/ingress-http-multiple-services.golden

```diff
@@ -0,0 +1,85 @@
+{
+  "versionInfo": "00000001",
+  "resources": [
+    {
+      "@type": "type.googleapis.com/envoy.api.v2.Listener",
+      "name": "http:1.2.3.4:443",
+      "address": {
+        "socketAddress": {
+          "address": "1.2.3.4",
+          "portValue": 443
+        }
+      },
+      "filterChains": [
+        {
+          "filters": [
+            {
+              "name": "envoy.http_connection_manager",
+              "config": {
+                  "http_filters": [
+                        {
+                              "name": "envoy.router"
+                            }
+                      ],
+                  "rds": {
+                        "config_source": {
+                              "ads": {
+                                  }
+                            },
+                        "route_config_name": "http_443"
+                      },
+                  "stat_prefix": "ingress_upstream_http_443_http",
```

Comment:
The way this name gets generated is a little weird (we don't have a real name for the listener besides the port/proto combo) which is why it ended up this way - we can change it to be `_443_http` instead of `_http_443_http` though.

Reply:
done